README.md

```diff
@@ -14,12 +14,13 @@ Once SDX calls this gateway, case receipts are then published to the RabbitMQ qu
 
 ## Running
 
-There are two ways of running this service.
-The easiest way is via docker (https://github.com/ONSdigital/ras-rm-docker-dev)
+There are two ways of testing this service. The easiest way is to deploy it to your sandbox, using Github Actions.
 
-Alternatively, you can run the service locally. For this, you'll need redis server running in a second window (port is currently set to 7379). 
+It can also be deployed in a local container using docker-dev: (https://github.com/ONSdigital/ras-rm-docker-dev).
+
+For this, you'll need a redis server running in a second window (port is currently set to 7379). 
 ```bash
-redis-server /usr/local/etc/redis.conf
+$ redis-server /usr/local/etc/redis.conf
```

Comment:
was this intentional?

Reply:
Yes, it's just showing the bash prompt for consistency with the other formatted bash